specification/batch/resource-manager/Microsoft.Batch/stable/2021-01-01/BatchManagement.json

```diff
@@ -0,0 +1,5332 @@
+          },
+          "BatchAccountCreate_SystemAssignedIdentity": {
+            "$ref": "./examples/BatchAccountCreate_SystemAssignedIdentity.json"
+          },
+          "BatchAccountCreate_UserAssignedIdentity": {
+            "$ref": "./examples/BatchAccountCreate_UserAssignedIdentity.json"
+          }
+        },
+        "description": "Creates a new Batch account with the specified parameters. Existing accounts cannot be updated with this API and should instead be updated with the Update Batch Account API.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "accountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z0-9]+$",
+            "minLength": 3,
+            "maxLength": 24,
+            "description": "A name for the Batch account which must be unique within the region. Batch account names must be between 3 and 24 characters in length and must use only numbers and lowercase letters. This name is used as part of the DNS name that is used to access the Batch service in the region in which the account is created. For example: http://accountname.region.batch.azure.com/."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BatchAccountCreateParameters"
+            },
+            "description": "Additional parameters for account creation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the Batch account entity.",
+            "schema": {
+              "$ref": "#/definitions/BatchAccount"
+            }
+          },
+          "202": {
+            "description": "The operation will be completed asynchronously.",
+            "headers": {
+              "Location": {
+                "description": "The URL of the resource used to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "Suggested delay to check the status of the asynchronous operation. The value is an integer that specifies the delay in seconds.",
+                "type": "integer",
+                "format": "int32"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
`x-ms-long-running-operation-options` should also specified to indicate Location header is used for tracking.
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation

Reply:
Added `x-ms-long-running-operation-options` for all long running operations. All are using `"location"` except one PrivateEndpointConnection operation which uses `"azure-async-operation"`